lib/zendesk_apps_support/package.rb

```diff
@@ -282,23 +264,25 @@ def app_js
     end
 
     def location_icons
-      {}.tap do |location_icons|
-        manifest.locations.each do |host, locations_for_host|
-          locations_for_host.keys.each do |location|
-            next unless LOCATIONS_WITH_ICONS.include?(location)
-            location_icons[host] ||= {}
-            location_icons[host][location] = if (has_file?("assets/icon_#{location}.svg"))
-              cache_busting_param = "?#{Time.now.to_i}" unless @is_cached
-              { 'svg' => "icon_#{location}.svg#{cache_busting_param}" }
-            elsif (has_file?("assets/icon_#{location}_inactive.png"))
-              {
-                'inactive' => "icon_#{location}_inactive.png",
-                'active' => has_file?("assets/icon_#{location}_active.png") ? "icon_#{location}_active.png" : "icon_#{location}_inactive.png",
-                'hover' => has_file?("assets/icon_#{location}_hover.png") ? "icon_#{location}_hover.png" : "icon_#{location}_inactive.png"
-              }
-            else
-              {}
-            end
+      Hash.new { |h, k| h[k] = {} }.tap do |location_icons|
```

Comment:
It did make me have to go look for docs though, wondering if it might be better to explicitly assign the value each time?